### PR TITLE
Added links for Privacy Policy and TOS

### PR DESCRIPTION
**Pull request by AleshaOleg**

### CODE REVIEW

**Review by sk7 — COMMENTED** [review]
the urls for both items are dynamic and come from the server in some RPC call. the urls are different for Untis for example.

File: src/components/org-info/menuItems.js [hunk 1]
```diff
@@ -83,6 +83,34 @@ export const SupportItem = ({ href }) => (
   </Item>
 )
 
+export const PrivacyPolicyItem = () => (
+  <Item
+    icon="doorLock"
+    href="https://www.grape.io/privacy"
```
Comment (sk7) [inline src/components/org-info/menuItems.js:89]: here

File: src/components/org-info/menuItems.js [hunk 2]
```diff
@@ -83,6 +83,34 @@ export const SupportItem = ({ href }) => (
...
+      defaultMessage="Privacy Policy"
+      description="Main org dropdown menu"
+    />
+  </Item>
+)
+
+export const TermsOfServiceItem = () => (
+  <Item
+    icon="bookBookmarked"
+    href="https://www.grape.io/terms-of-service"
```
Comment (sk7) [inline src/components/org-info/menuItems.js:103]: and here

**Review by sk7 — CHANGES_REQUESTED** [review]

File: src/components/org-info/menuItems.js (hunk 1 above)
Comment (AleshaOleg) [reply src/components/org-info/menuItems.js]: thanks, fixed it.

File: src/components/org-info/menuItems.js (hunk 2 above)
Comment (AleshaOleg) [reply src/components/org-info/menuItems.js]: thanks

**Pull request merged**